pkg/controller/util/handlers.go

```diff
@@ -100,10 +44,7 @@ func NewTriggerOnMetaAndFieldChanges(field string, triggerFunc func(pkgruntime.O
 		},
 		UpdateFunc: func(old, cur interface{}) {
 			curObj := cur.(pkgruntime.Object)
-			oldMeta := getFieldOrPanic(old, "ObjectMeta").(metav1.ObjectMeta)
```

Comment:
As per a previous comment, please remove `NewTriggerOnMetaAndSpecChanges`
 and `NewTriggerOnMetaAndFieldChanges` rather than updating them.  There are no non-test clients, so continuing to maintain the code has zero value.

Reply:
i have removed them.
i guess it's just `diff -p` being confused by the removal.

Reply:
Ah, my apologies.  As much as I'd like to blame the tool, I think I'm the one who's confused.